commit 7020e84ab9a6cc44e8167bdb144822d7dfc97a38
Author: Dominique Leuenberger <dimstar@opensuse.org>
Date:   Thu Feb 27 14:54:13 2020 +0100

    build: Fix build on systems with wayland-client headers in non-default location
    
    Add a missing dependency to wl_client_dep for the wayland build. Some distros
    have the wayland-client headers not installed in /usr/include (which is perfectly
    valid, the pkg-config .pc file gives the right feedback).

diff --git a/gst-libs/gst/wayland/meson.build b/gst-libs/gst/wayland/meson.build
index b1ede41f1..89bc6ab57 100644
--- a/gst-libs/gst/wayland/meson.build
+++ b/gst-libs/gst/wayland/meson.build
@@ -15,7 +15,7 @@ if use_wayland
     soversion : soversion,
   darwin_versions : osxversion,
     install : true,
-    dependencies : [gst_dep, gstvideo_dep]
+    dependencies : [gst_dep, gstvideo_dep, wl_client_dep]
   )
 
   gstwayland_dep = declare_dependency(link_with : gstwayland,
